README.md

```diff
@@ -0,0 +1,222 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
```

Comment:
- [ ] `Required`
- [ ] Please Kindly add a **Live version** of your
- [ ]  It should be deployed using Heroku, Netlify or GitHub Pages.

Reply:
* The Live demo link isn't working as pointed out by the previous CR, Kindly fix this 🙏 
![image](https://user-images.githubusercontent.com/52497246/221391034-81934e4b-6a15-4672-9ec3-bfe6f239917e.png)